packages/core/config/webpack.common.js

```diff
@@ -148,6 +148,8 @@ const getSharedPlugins = (isEnvProduction) => {
     // time.
     new webpack.DefinePlugin({
       __REDWOOD__API_PROXY_PATH: JSON.stringify(redwoodConfig.web.apiProxyPath),
+      __REDWOOD__APP_TITLE:
+        redwoodConfig.web.title || path.basename(redwoodPaths.base),
```

Comment:
Whoops, I left a comment, but forgot to finish review.


```suggestion
        JSON.stringify(redwoodConfig.web.title) || path.basename(redwoodPaths.base),
```

Should fix it!